packages/cardano-services/test/ChainHistory/DbSyncChainHistoryProvider/ChainHistoryBuilder.test.ts

```diff
@@ -3,7 +3,30 @@ import { ChainHistoryBuilder } from '../../../src';
 import { ChainHistoryFixtureBuilder, TxWith } from '../fixtures/FixtureBuilder';
 import { DataMocks } from '../../data-mocks';
 import { Pool } from 'pg';
-import { logger } from '@cardano-sdk/util-dev';
+import { createLogger } from '@cardano-sdk/util-dev';
+
+const logger = createLogger({ record: true });
+
+/**
+ * Checks tx_ids are logged as hex strings
+ *
+ * @param text the text of the loggd message containing tx_ids
+ */
+const checkLoggedTxids = (text: string) => {
```

Comment:
```suggestion
const assertLoggedTxIds = (text: string): void => {
```

Reply:
This was inferred by the absence of a `return` statement, but better to make it explicit.
ed6416dc7 should solve this